samples/matter/common/src/watchdog/Kconfig

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig NCS_SAMPLE_MATTER_WATCHDOG
+	bool "Watchdog for Matter applications"
+	select WATCHDOG
+
+if NCS_SAMPLE_MATTER_WATCHDOG
+
+config NCS_SAMPLE_MATTER_WATCHDOG_DEFAULT
+    bool "Use watchdog object dedicated for Main and Matter threads"
+    default y
+    help
+        The default watchdog objects are created in the matter_init.cpp file.
+        The watchdog objects are dedicated for Main and Matter threads and are initialized
+        using the NCS_SAMPLE_MATTER_WATCHDOG_DEFAULT_FEED_TIME value.
+
+config NCS_SAMPLE_MATTER_WATCHDOG_TIMEOUT
+    int "Default maximum window time in milliseconds for receiving the feeding signal"
+    default 10000
+    help
+        The default maximum window time defines a time within which the feeding signal must be received
+        from all created Watchdog sources to reset the watchdog object's timer.
+        If the signals come after elapsing this time, the watchdog timer will not be reset and the
+        reboot occur.
+
+config NCS_SAMPLE_MATTER_WATCHDOG_DEFAULT_FEED_TIME
+    int "Default interval of feeding time in milliseconds"
+    default 1000
```

Comment:
This seems quite low from power power consumption perspective. Shouldn't we rather deduce the interval from the timeout?  If timeout is, say, a minute, then feeding the watchdog every 30 seconds should be enough?

Reply:
Yes, that's right. I checked that, and a 1-second interval causes ~1uA higher power consumption. If we change it to 30 seconds and timeout to 1 minute it probably will have an insignificant impact on power consumption.

Reply:
And deducing this value from the timeout seems good.